lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -1,17 +1,24 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE BlockArguments #-}
```

Comment:
I have the feeling that `BlockArguments` does affect our shared Haskell idiom, so it would be swell if we were to discuss it as a team — and then put it in the .cabal file.

Reply:
Yes, we have it on the team discussion agenda for some time already, once its agreed - I'll move it to `cabal` file.